less-print/viewdetails.less

```diff
@@ -166,3 +170,19 @@ h3 {
   font-weight: bold;
   padding: 5px 0 5px 0;
 }
+
+.route-activity::before {
+  content:attr(uib-tooltip);
+  font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-size:10.5px;
```

Comment:
A float value for a pixel size? WHat's the interest?

Reply:
I didn't think a lot, it was the value for the other informations.
In fact, it's 0.75em for the other informations. I need to put this one to 0.35em if I want the same rendering (don't know why). But this way is better than px, that's right, I change that.